[2023-05-16T09:54:35.167+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: my_third_dagV3.fo manual__2023-05-16T09:51:15.585964+00:00 [queued]>
[2023-05-16T09:54:35.219+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: my_third_dagV3.fo manual__2023-05-16T09:51:15.585964+00:00 [queued]>
[2023-05-16T09:54:35.219+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-05-16T09:54:35.219+0000] {taskinstance.py:1280} INFO - Starting attempt 2 of 2
[2023-05-16T09:54:35.219+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-05-16T09:54:35.247+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): fo> on 2023-05-16 09:51:15.585964+00:00
[2023-05-16T09:54:35.255+0000] {standard_task_runner.py:55} INFO - Started process 1126 to run task
[2023-05-16T09:54:35.260+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'my_third_dagV3', 'fo', 'manual__2023-05-16T09:51:15.585964+00:00', '--job-id', '143', '--raw', '--subdir', 'DAGS_FOLDER/dag3.py', '--cfg-path', '/tmp/tmp9o93qraz']
[2023-05-16T09:54:35.261+0000] {standard_task_runner.py:83} INFO - Job 143: Subtask fo
[2023-05-16T09:54:35.526+0000] {task_command.py:388} INFO - Running <TaskInstance: my_third_dagV3.fo manual__2023-05-16T09:51:15.585964+00:00 [running]> on host d1c47018cb38
[2023-05-16T09:54:35.630+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_third_dagV3
AIRFLOW_CTX_TASK_ID=fo
AIRFLOW_CTX_EXECUTION_DATE=2023-05-16T09:51:15.585964+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-16T09:51:15.585964+00:00
[2023-05-16T09:54:58.971+0000] {logging_mixin.py:137} INFO - Hello
[2023-05-16T09:54:58.971+0000] {logging_mixin.py:137} INFO - <pyspark.sql.session.SparkSession object at 0x7fcf2a4f3590>
[2023-05-16T09:54:58.971+0000] {logging_mixin.py:137} INFO - Bye
[2023-05-16T09:55:08.211+0000] {logging_mixin.py:137} INFO - root
 |-- ph_id: integer (nullable = true)
 |-- date: date (nullable = true)
 |-- price: float (nullable = true)
 |-- sales_qty_base: float (nullable = true)
 |-- sales_qty: integer (nullable = true)
 |-- store_location_id: integer (nullable = true)
 |-- product_id: integer (nullable = true)
 |-- store_location_geo_clust: integer (nullable = true)
 |-- product_cluster_id: integer (nullable = true)
[2023-05-16T09:55:08.211+0000] {python.py:177} INFO - Done. Returned value was: 2
[2023-05-16T09:55:08.343+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=my_third_dagV3, task_id=fo, execution_date=20230516T095115, start_date=20230516T095435, end_date=20230516T095508
[2023-05-16T09:55:08.475+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-05-16T09:55:08.506+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
